webapp/src/main/java/org/opentestsystem/rdw/admin/service/CsvValidationService.java

```diff
@@ -18,10 +18,10 @@
      * Validate the given CSV's headers and return the collection
      * of validation failures.
      *
-     * @param csvHeaders CSV headers
+     * @param csvHeaders The map of CSV header to index
      * @return The list of validation failures, empty if valid
      */
-    List<CsvValidationFailure> validateHeaders(Set<String> csvHeaders);
+    List<CsvValidationFailure> validateHeaders(Map<String, Integer> csvHeaders);
```

Comment:
Uhm, why not pass a List<String> in? A list is ordered.

Reply:
I suppose I could, and do the map-to-list conversion in the client.  The problem is that the CSVParser only gives me a map, so I have to do the map-to-list conversion somewhere.

Reply:
changed